examples/using-gatsby-image/src/components/navigation.js

```diff
@@ -17,46 +21,63 @@ const Navigation = () => (
         Gatsby Image
       </Link>
     </h1>
-    <nav
+    <ul
       css={{
         fontFamily: options.headerFontFamily.join(`,`),
-        "& a": {
-          color: options.bodyColor,
-          opacity: 0.7,
-          textTransform: `uppercase`,
-          backgroundImage: `none`,
+        margin: 0,
+        listStyle: `none`,
+        "& li": {
```

Comment:
Any reason not to just put these on the component directly?

Reply:
None but trying to be fast – and being sloppy in consequence.
Will fix!